config/routes.rb

```diff
@@ -10,41 +10,100 @@
   # as per Loc/Id definition
 
   # Special (/ref-based) Loc/Id routes
+  specified_get '/ref/:reference/:repository_id/*locid' => 'api/v1/ontology_versions#show',
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.